packages/machine/src/models/state-channel.ts

```diff
@@ -115,7 +117,7 @@ export class StateChannel {
     return this.appInstances.get(idx!)!;
   }
 
-  public setupChannel(network: NetworkContext) {
+  public setupChannel(network: NetworkContext): StateChannel {
```

Comment:
I don't think we need this here

Reply:
The function returns a `StateChannel` instance. Not having that in the function signature to indicate the return type is misleading, as one would think the function returns void.

Reply:
It would be implicitly caught by the type checker

Reply:
Of course, my point was for more targeted towards legibility.